### PR TITLE
add: Thanos metrics retention rules

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by axeoman — APPROVED** [review]

File: docs/products/metrics/concepts/retention-rules.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Retention rules
+---
+Retention periods in Aiven for Metrics® help you manage how long your metrics data is stored. Understanding how to set these periods allows you to optimize storage space while maintaining access to historical data for analysis.
+
+## Understand retention rules
+
+Retention rules define the duration for which Aiven for Metrics stores your metrics data.
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:8]: There's a lilttle bit of repeat between this paragraph and the first one.  "help you manage how long your metrics data is stored." is here as "the duration for which Aiven for Metrics stores your metrics data". 

Can we remove the repetition? Also do we need the "understand retention rules" title? I see limited seo or readbility benefits, and this probably pushes for repetion, why not just have a couple of paragraphs directly under the main title?

File: docs/products/metrics/concepts/retention-rules.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+historical insights. However, tailoring these settings allows you to align with
+specific data storage strategies, compliance needs, and cost management goals.
+
+## Define retention rules
+
+Aiven for Metrics uses the Thanos Metrics Compactor to simplify retention settings.
+A single parameter, `compactor.retention.days`, sets the same retention period for all
+types of data: `raw`, 5`-minute downsampled`, and `1-hour downsampled`.
+
+To adjust the `compactor.retention.days` parameter
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:19]:
```suggestion
To adjust the `compactor.retention.days` parameter:
```

File: docs/products/metrics/concepts/retention-rules.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Click** Service settings** from the sidebar.
+1. Scroll to **Advanced configuration** and click **Configure**.
+1. Click **Add configuration options**.
+1. Locate the `compactor.retention.days` parameter.
+1. Set the desired retention period by adjusting the parameter value.
+1. Click **Save configuration**.
+
+
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:28]:
```suggestion
```

File: docs/products/metrics/concepts/retention-rules.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+## Downsampling
+
+
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:31]:
```suggestion
```

File: docs/products/metrics/concepts/retention-rules.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+1. Click** Service settings** from the sidebar.
+1. Scroll to **Advanced configuration** and click **Configure**.
+1. Click **Add configuration options**.
+1. Locate the `compactor.retention.days` parameter.
+1. Set the desired retention period by adjusting the parameter value.
+1. Click **Save configuration**.
+
+
+## Downsampling
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:29]: Here I suspect we might get SEO benefit by being more specific? `Data series downsampling` or something similar perhaps?

File: docs/products/metrics/concepts/retention-rules.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+
+Downsampling in Aiven for Metrics transforms data series to lower resolution without
+compromising accuracy. This technique enhances the performance of queries over large
+time frames. Aiven for Metrics systematically aggregates data into resolutions of
+5 minutes and 1 hour. This aggregation process significantly improves query performance
+over extended periods and increases storage efficiency.
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:36]: `This technique enhances the performance of queries over large time frames.` seems to mean the same thing as `This aggregation process significantly improves query performance over extended periods`, could we simplify?

File: sidebars.ts [hunk 7]
```diff
@@ -1493,7 +1493,9 @@ const sidebars: SidebarsConfig = {
               },
               items: [
                 'products/metrics/concepts/metrics-overview',
+                'products/metrics/concepts/retention-rules',
                 'products/metrics/concepts/storage-resource-scaling',
+
```
Comment (ArthurFlag) [inline sidebars.ts:1498]:
```suggestion
```

**Review by ArthurFlag — CHANGES_REQUESTED** [review]

File: docs/products/metrics/concepts/retention-rules.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+
```
Comment (ArthurFlag) [inline docs/products/metrics/concepts/retention-rules.md:38]:
```suggestion
```

File: docs/products/metrics/concepts/retention-rules.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
+---
+title: Retention rules
+---
+Retention periods in Aiven for Metrics® help you manage how long your metrics data is stored. Understanding how to set these periods allows you to optimize storage space while maintaining access to historical data for analysis.
```
[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/products/metrics/concepts/retention-rules.md]: Good catch. It is retention rules and not retention periods. Fixed this.

File: docs/products/metrics/concepts/retention-rules.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/metrics/concepts/retention-rules.md]: Update the cotnent.

File: docs/products/metrics/concepts/retention-rules.md (hunk 5 above)
Comment (harshini-rangaswamy) [reply docs/products/metrics/concepts/retention-rules.md]: Updated subheading

**Review by ArthurFlag — APPROVED** [review]
Thanks! 👍

**Pull request merged**